app/helpers/report_child_abuse_helper.rb

```diff
@@ -0,0 +1,11 @@
+module ReportChildAbuseHelper
```

Comment:
Could this be `PhoneNumberHelper` ?  I know it's only used in the report child abuse partial for now, but maybe it can be used elsewhere?  I guess there's a line to walk around how custom this regexp parsing is for the exact data we know we're getting from this dataset.